src/scss/_custom.scss

```diff
@@ -182,14 +182,22 @@ input[type="password"]:not(:placeholder-shown) {
   position: relative;
 }
 
-.input-password .eye {
+.input-password .span {
   position: absolute;
   right: 20px;
   z-index: 1000;
   top: 4px;
   cursor: pointer;
 }
 
+.input-password .eye {
+  position: absolute;
+  right: 20px;
+  z-index: 1000;
+  top: 10px;
+  cursor: pointer;
+}
+
```

Comment:
Disrupts all eyes in the inputs field. Has to stay as before.
![photo1678588104](https://user-images.githubusercontent.com/90323042/224520804-b8f816e4-7e3d-4817-8de0-40a9bcf67cef.jpeg)